### ISSUE TITLE
Unable to realise the effect of the article

### ISSUE THREAD

**Issue by A1475465724** [VinAIResearch/Blur2Blur#5]
Tested using GOPRO and RED data, set up and trained according to the parameters of the paper, the final results show that the effect in the paper cannot be achieved

**Comment by ZhijingS** [comment]
May I ask how do you split the image set B and S in the paper? Do the scenes in image sets B and S not overlap at all, or will images of the same scene be partly in B and partly in S？Looking forward your reply.

**Comment by A1475465724** [comment]
> May I ask how do you split the image set B and S in the paper? Do the scenes in image sets B and S not overlap at all, or will images of the same scene be partly in B and partly in S？Looking forward your reply.

**Comment by ZhijingS** [comment]
Thanks for your reply! I think if the scenes in B and S overlap, it may be easier for the blur translater to learn how to transfer blur kernel and may get better results. However I haven't try it.

**Comment by A1475465724** [comment]
> Thanks for your reply! I think if the scenes in B and S overlap, it may be easier for the blur translater to learn how to transfer blur kernel and may get better results. However I haven't try it.

Thank you for your advice，Have you try this paper's code? what  is your result? Does this article's theory really work?

**Comment by ZhijingS** [comment]
No, I haven't tried the code. I just take the idea of the paper and try to make a unsupervised deblur algorithm. Sadly, my experiments didn't work.

**Comment by A1475465724** [comment]
> No, I haven't tried the code. I just take the idea of the paper and try to make a unsupervised deblur algorithm. Sadly, my experiments didn't work.

I have tried this paper's code.However , the results showed it didn‘t work,Maybe you can try other ways

**Comment by ZhijingS** [comment]
Thanks for your advice. Do you focus on unsupervised deblur algorithms? I think most of them need a GAN to try to create more realistic blur and then we can deblur, however the lack of a good metric for realistic blur makes it difficult to guide GAN training.
